### DISCUSSION TITLE
Storybook Vue3 | Laravel | Webpack error

### DISCUSSION THREAD

**Discussion by alex129** [storybookjs/storybook#20557] [Help]
Hello! 

I've installed storybook in my current proyect. I'm using webpack and vue 3 with Laravel. The installation was correct but when I run ``
npm run storybook' and I open the page, the following error is shown:

![image](https://user-images.githubusercontent.com/10452211/211508467-942d54a1-b836-4c2d-9527-12d08e675771.png)

Anyone can help me?

Thanks

**Comment by chakAs3** [comment]
With Laravel ????  it is bit challenging, I think Storybook should run on pure js project, even with nuxt it is quite tricky to make it run properly
